src/ppmseq/ugbio_ppmseq/ppmSeq_utils.py

```diff
@@ -902,7 +902,9 @@ def read_trimmer_failure_codes_ppmseq(trimmer_failure_codes_csv: str):
     AssertionError
         If the columns are not as expected
     """
-    df_trimmer_failure_codes = read_trimmer_failure_codes(trimmer_failure_codes_csv)
+    df_trimmer_failure_codes = read_trimmer_failure_codes(
+        trimmer_failure_codes_csv, include_failed_rsq=False, add_total=True
```

Comment:
Just to understand: now rsq filtering is done by trimmer. you can either include the rsq filtering information in the trimmer failure code file or not. default is not to include it. So how can I know hoe many reads were filtered for rsq?

Reply:
This is not a metric we ever tracked in the application QC